### DISCUSSION TITLE
Restore state of sensor node

### DISCUSSION THREAD

**Discussion by sheminasalam** [zachowj/hass-node-red#158] [Q&A] [answered]
I am using the sensor node to count a specific process but the issue I am facing is that when HA server is restarted, the sensor loses value. How I can make sure that the sensor restores value even after HA server restart? I believe in the older entity type node, it was restoring state but not in latest 4 nodes.

**Comment by zachowj** [comment, marked as answer]
It's still there, just moved from the entity node to the entity config.

**Reply by sheminasalam** [reply]
I just enabled Resend state and attributes in config and everything works as intended. Thanks
